### ISSUE TITLE
Crash on program start in 0.4.0 version

### ISSUE THREAD

**Issue by kedarac** [dancol90/mi-360#8]
@dancol90 First off thank you for the update 0.4.

I seem to have a crash everytime I start mi-360. Here is the error screenshot
![capture](https://user-images.githubusercontent.com/17289426/53743241-5a3b7600-3ec0-11e9-9e6c-1aef09aface8.PNG)

**Comment by kedarac** [comment]
I solved this by building from source and using the mi-360.exe and HidLibrary.dll from there. But I still do not get why it could not access the library mentioned in the above screenshot when using the installer.

**Comment by dancol90** [comment]
Hi, 
That's strange, because all the dependencies are packed with the main executable.
Have you tried downloading the setup again and reinstall it? Maybe it got corrupt

**Comment by kedarac** [comment]
Yes, I did try that but it did not solve the issue. The weird thing is the mi-360.exe and HidLibrary.dll that I built from source throw the same error when I placed them(overwrote existing files) in the installation directory of mi-360 i.e: C:\Program Files\mi-360. But it seems to work when I run mi-360.exe from a different location without the Nefarius.ViGEmClient.dll file being in the same directory

Below screenshot shows the folder structure of the files built from source. I run the mi-360.exe directly from here and it works.

![Capture](https://user-images.githubusercontent.com/17289426/56550748-21587d00-65a4-11e9-9866-c6b22bea0950.PNG)
.
